packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -279,68 +314,92 @@ function SignUpStartInternal(): JSX.Element {
-            />
-          </Card.Action>
-        </Card.Footer>
-      </Card.Root>
+      {!alternativePhoneCodeProvider ? (
+        <Card.Root>
+          <Card.Content>
+            <Header.Root showLogo>
+              <Header.Title
+                localizationKey={
+                  isCombinedFlow
+                    ? localizationKeys('signUp.start.titleCombined')
+                    : localizationKeys('signUp.start.title')
+                }
+              />
+              <Header.Subtitle
+                localizationKey={
+                  isCombinedFlow
+                    ? localizationKeys('signUp.start.subtitleCombined')
+                    : localizationKeys('signUp.start.subtitle')
+                }
+              />
+            </Header.Root>
+            <Card.Alert>{card.error}</Card.Alert>
+            <Flex
+              direction='col'
+              elementDescriptor={descriptors.main}
+              gap={6}
+            >
+              <SocialButtonsReversibleContainerWithDivider>
+                {(showOauthProviders || showWeb3Providers) && (
+                  <SignUpSocialButtons
+                    enableOAuthProviders={showOauthProviders}
+                    enableWeb3Providers={showWeb3Providers}
+                    enableAlternativePhoneCodeProviders={showAlternativePhoneCodeProviders}
+                    onAlternativePhoneCodeProviderClick={onAlternativePhoneCodeProviderClick}
+                    continueSignUp={missingRequirementsWithTicket}
+                    legalAccepted={Boolean(formState.legalAccepted.checked) || undefined}
+                  />
+                )}
+                {shouldShowForm && (
+                  <SignUpForm
+                    handleSubmit={handleSubmit}
+                    fields={fields}
+                    formState={formState}
+                    canToggleEmailPhone={canToggleEmailPhone}
+                    handleEmailPhoneToggle={handleChangeActive}
+                  />
+                )}
+              </SocialButtonsReversibleContainerWithDivider>
+              {!shouldShowForm && <CaptchaElement />}
+            </Flex>
+          </Card.Content>
+
+          <Card.Footer>
+            <Card.Action elementId='signUp'>
+              <Card.ActionText localizationKey={localizationKeys('signUp.start.actionText')} />
+              <Card.ActionLink
+                localizationKey={localizationKeys('signUp.start.actionLink')}
+                to={isCombinedFlow ? '../' : clerk.buildUrlWithAuth(signInUrl)}
+              />
+            </Card.Action>
+          </Card.Footer>
+        </Card.Root>
+      ) : (
+        <SignUpAlternativePhoneCodePhoneNumberCard
+          handleSubmit={handleAlternativePhoneCodeSubmit}
+          fields={fields}
+          formState={formState}
+          onUseAnotherMethod={onAlternativePhoneCodeUseAnotherMethod}
+          phoneCodeProvider={alternativePhoneCodeProvider}
+        />
+      )}
```

Comment:
This is the only thing that is actually added on JSX, the rest above is just indentation changes.